src/main/java/randoop/main/Minimize.java

```diff
@@ -161,23 +160,21 @@ public boolean handle(String[] args) throws RandoopTextuiException {
       }
     } catch (Options.ArgException ae) {
       usage("while parsing command-line arguments: %s", ae.getMessage());
+      throw new RandoopTextuiException("See usage.");
     }
 
     if (Minimize.suitepath == null) {
-      System.out.println("You must specify an input file path.");
-      System.out.println("Use the --suitepath option.");
-      System.exit(1);
+      throw new RandoopTextuiException(
+          "You must specify an input file path to the 'minimize' command. Use the --suitepath option.");
     }
 
     // Check that the input file is a Java file.
     if (!FilenameUtils.getExtension(Minimize.suitepath).equals("java")) {
-      System.out.println("The input file must be a Java file.");
-      System.exit(1);
+      throw new RandoopTextuiException("The input file must be a Java file.");
```

Comment:
The error message would be more useful if it also said what the file name was.
The user probably got here via bad cut-and-paste or a similar problem that might not be obvious from staring at the Randoop command line.

The same comment goes for timeout below, but it isn't as critical there.